main.tf

```
@@ -92,6 +86,11 @@ resource "random_id" "uniq" {
 // Only create global resources if global variable is set to true
 // count = var.global ? 1 : 0
 
+resource "lacework_external_id" "aws_iam_external_id" {
+  csp        = "aws"
+  account_id = local.lacework_account
```

This should NOT be the Lacework AWS account, but instead, the account that is being integrated from the customer.

We can use https://registry.terraform.io/providers/hashicorp/aws/4.60.0/docs/data-sources/caller_identity
```suggestion
  account_id = data. aws_caller_identity.current.account_id
```